misc/python3/depgraph.sh

```diff
@@ -10,7 +10,7 @@ cd tahoe-depgraph
 # Generate the maybe-changed data.
 python "${TAHOE}"/misc/python3/tahoe-depgraph.py "${TAHOE}"
 
-if git diff-index --quiet HEAD; then
+if git diff-index --quiet HEAD tahoe-deps.json tahoe-ported.json; then
```

Comment:
I'm not sure, either. I saw an error from the depgraph CI workflow, and I hypothesized that this would resolve it. I only saw this error once and I haven't tried hard to reproduce it. I can try some more.